Chapter 7/Enemy.cs

```diff
@@ -25,8 +25,6 @@ public override IEnumerator DamageCharacter(int damage, float interval)
         {
             hitPoints = hitPoints - damage;
 
-            StartCoroutine(FlickerCharacter());
```

Comment:
Ah yes - I know exactly how this was introduced too. *kicks self* thank you for pointing out.